tests/Traits/BulkUserCreationGeneratedFileRemovalTrait.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ *  This program is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU General Public License
+ *  as published by the Free Software Foundation; under version 2
+ *  of the License (non-upgradable).
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ *  Copyright (c) 2021 (original work) Open Assessment Technologies S.A.
+ */
+
+declare(strict_types=1);
+
+namespace OAT\SimpleRoster\Tests\Traits;
+
+use Symfony\Component\Filesystem\Filesystem;
+
+trait BulkUserCreationGeneratedFileRemovalTrait
+{
+    protected function removeGeneratedUsersFilePath(): void
+    {
+        $filesystem = new Filesystem();
```

Comment:
Maybe worth check how Dani worked with filesystem in the past, you can check these:

https://github.com/oat-sa/simple-roster/blob/develop/src/Csv/CsvReaderBuilder.php#L30
https://github.com/oat-sa/simple-roster/blob/develop/src/Storage/StorageRegistry.php

Maybe it can become a separated ticket